ui/src/components/NavigationBar/NavBar.js

```diff
@@ -0,0 +1,233 @@
+import React, { useState } from "react";
+import {
+  Flex,
+  Center,
+  Stack,
+  Box,
+  Text,
+  Image,
+  Button,
+  Menu,
+  Container,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Drawer,
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  useDisclosure,
+  Img,
+} from "@chakra-ui/react";
+import { Link } from "react-router-dom";
+import MainMenu from "./MainMenu";
+
+const Navigationbar = () => {
```

Comment:
use PascalCase properly for class names. The name of your file should also correspond to the name of your component. This should be called `NavBar`

Reply:
fixed this